pwndbg/commands/__init__.py

```diff
@@ -259,7 +259,7 @@ def _OnlyAmd64(*a, **kw):
 def OnlyWithLibcDebugSyms(function):
```

Comment:
We should rename it to `OnlyWithResolvedHeapSyms` or `OnlyWithHeap` or some other name if u have a better one

Reply:
I agree with yours.
I think `OnlyWithResolvedHeapSyms` is good, I'll use it.